homeassistant/components/metoffice/sensor.py

```diff
@@ -95,15 +105,18 @@ async def async_setup_entry(
 class MetOfficeCurrentSensor(Entity):
     """Implementation of a Met Office current weather condition sensor."""
 
-    def __init__(self, entry_data, hass_data, sensor_type):
+    def __init__(self, entry_data, hass_data, use_3hourly, sensor_type):
         """Initialize the sensor."""
         self._data = hass_data[METOFFICE_DATA]
         self._coordinator = hass_data[METOFFICE_COORDINATOR]
 
         self._type = sensor_type
-        self._name = f"{hass_data[METOFFICE_NAME]} {SENSOR_TYPES[self._type][0]}"
+        self._name = f"{hass_data[METOFFICE_NAME]} {SENSOR_TYPES[self._type][0]} {MODE_3HOURLY_LABEL if use_3hourly else MODE_DAILY_LABEL}"
         self._unique_id = f"{SENSOR_TYPES[self._type][0]}_{self._data.latitude}_{self._data.longitude}"
+        if not use_3hourly:
```

Comment:
Since we lost the comments on the previous review, and so I can remember, the currently existing entity is the 3hourly one?

Reply:
It's written right in the description above, ignore this then.